api-flask/app/database/alert_database.py

```diff
@@ -31,7 +30,7 @@ def __init__(self):
         """Alerts Database initializer."""
         _eng = create_engine(DB_URI)
         self.session = sessionmaker(_eng)()
-        logger.info('DB connection is initialized.')
+        logger.info('DB connection is initialized, database: {}.'.format(getenv('POSTGRES_DB')))
```

Comment:
Let's keep it as you had it. Because the DB info can come from DATABASE_URL this might be undefined anyways.

Reply:
Removed.